solutions/security/get-started/enable-threat-intelligence-integrations.md

```diff
@@ -19,4 +19,78 @@ $$$agent-ti-integration$$$
 
 $$$custom-ti-integration$$$
 
-$$$ti-mod-integration$$$
\ No newline at end of file
+$$$ti-mod-integration$$$
+
+
+The Threat Intelligence view provides a streamlined way to collect threat intelligence data that you can use for threat detection and matching. Threat intelligence data consists of  [threat indicators](../../../troubleshoot/security/indicators-of-compromise.md#ti-indicators) ingested from third-party threat intelligence sources.
+
+Threat indicators describe potential threats, unusual behavior, or malicious activity on a network or in an environment. They are commonly used in indicator match rules to detect and match known threats. When an indicator match rule generates an alert, it includes information about the matched threat indicator.
+
+::::{note}
+To learn more about alerts with threat intelligence, visit [View alert details](../../../solutions/security/detect-and-alert/view-detection-alert-details.md).
+
+::::
+
+
+You can connect to threat intelligence sources using an [{{agent}} integration](../../../solutions/security/get-started/enable-threat-intelligence-integrations.md#agent-ti-integration), the [Threat Intel module](../../../solutions/security/get-started/enable-threat-intelligence-integrations.md#ti-mod-integration), or a [custom integration](../../../solutions/security/get-started/enable-threat-intelligence-integrations.md#custom-ti-integration).
+
+:::{image} ../../../images/serverless--getting-started-threat-intelligence-view.png
+:alt: The Threat Intelligence view on the Overview dashboard
+:class: screenshot
+:::
+
+There are a few scenarios when data won’t display in the Threat Intelligence view:
+
+* If you’ve chosen a time range that doesn’t contain threat indicator event data, you are prompted to choose a different range. Use the date and time picker in the {{security-app}} to select a new range to analyze.
+* If the {{agent}} or {{filebeat}} agent hasn’t ingested Threat Intel module data yet, the threat indicator event counts won’t load. You can wait for data to be ingested or reach out to your administrator for help resolving this.
+
+
+## Add an {{agent}} integration [agent-ti-integration]
+
+1. Install a [{{fleet}}-managed {{agent}}](https://www.elastic.co/guide/en/fleet/current/install-fleet-managed-elastic-agent.html) on the hosts you want to monitor.
+2. In the Threat Intelligence view, click **Enable sources** to view the Integrations page. Scroll down and select **Elastic Agent only** to filter by {{agent}} integrations.
+
+    ::::{tip}
+    If you know the name of {{agent}} integration you want to install, you can search for it directly. Alternatively, choose the **Threat Intelligence** category to display a list of available [threat intelligence integrations](https://docs.elastic.co/en/integrations/threat-intelligence-intro).
```

Comment:
The ref to integraiton docs might need to be updated to use [this attribute](https://github.com/elastic/docs/blob/ac5f90bfcf34e625d6eda1edb110b8766368201f/shared/attributes.asciidoc#L130).

Reply:
```suggestion
    If you know the name of {{agent}} integration you want to install, you can search for it directly. Alternatively, choose the **Threat Intelligence** category to display a list of available [threat intelligence integrations]({{integrations-docs}}/threat-intelligence-intro).
```